cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/01-school-search.html

```diff
@@ -0,0 +1,178 @@
+{% import 'atoms/radio-button.html' as radio %}
+
+<section class="college-costs_tool-section" id="college-costs_school-search" data-tool-section="school-info">
+    <div class="school-search_start">
+        <h2>School & living situation</h2>
+        <p>Start with the school you’re considering, as well as the program and where you expect to live.</p>
+        <label class="a-label a-label__heading u-mb15" for="search__school-input">
+            Search for a college:
+        </label>
+        <p>
+            <input type="text" 
+                   id="search__school-input"
+                   name="search__school-input"
+                   class="a-text-input"
+                   autocomplete="off">
+        </p>
+    </div>
+
+    <div id="search__results">
```

Comment:
Should this be `search-results`? This is saying there's a block `search`, that has a modifier "results", so for proper BEM this would be `search search__results`.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/9237cac212c52dfd381dd65958ae6f2af5f8570c